### ISSUE TITLE
The "Select" button runs the animation if a repository has already been added.

### ISSUE THREAD

**Issue by emanuelegurini** [Balastrong/myntenance#17]
The "Select" button should run the animation only if a repository does not already exist in the dashboard. 

Can we add the disable property to the button if a selected repository already exists?

**Comment by Balastrong** [comment]
Yep, we might also add a short, red text below explaining why the button is disabled.
